test/stampit-api.js

```diff
@@ -30,7 +30,7 @@ test('incorrect stampit({ refs }) args', function (t) {
   t.same(stampit({ refs: 42 }).fixed.refs, {});
   t.same(stampit({ refs: null }).fixed.refs, {});
   t.same(stampit({ refs: new RegExp() }).fixed.refs, {});
-  t.same(stampit({ refs: [42] }).fixed.refs, {});
+  t.same(stampit({ refs: [42] }).fixed.refs, { 0: 42 });
```

Comment:
I don't think my logic is wrong. `supermixer` and other mixin/merge functions iterates arrays the same as objects. So why should we treat an array as something special here?